generators/server/templates/entity/src/main/java/package/domain/Entity.java.jhi.jakarta_persistence.ejs

```diff
@@ -110,7 +110,7 @@ import org.hibernate.type.SqlTypes;
     }, allowSetters = true)
       <%_ } _%>
       <%_ if (relationship.ownerSide) { -%>
-    @OneToOne<% if (relationship.relationshipRequired) { %>(optional = false)<% } %>
+    @OneToOne(fetch = FetchType.LAZY<% if (relationship.relationshipRequired) { %>, optional = false<% } %>)
```

Comment:
This one is usually eager, but for the goodwill ^^

Reply:
I also thought think (see https://github.com/jhipster/generator-jhipster/pull/19791#issuecomment-1312003467), but this means other others relationships will load those relationships too.
IMO the safest right now it to make everything lazy.